### PR TITLE
init complete

### PR DESCRIPTION
**Pull request by rosnovsky**

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    // order to group specific specs to a specific capability.
+    //
+    // First, you can define how many instances should be started at the same time. Let's
+    // say you have 3 different capabilities (Chrome, Firefox, and Safari) and you have
+    // set maxInstances to 1; wdio will spawn 3 processes. Therefore, if you have 10 spec
+    // files and you set maxInstances to 10, all spec files will get tested at the same time
+    // and 30 processes will get spawned. The property handles how many capabilities
+    // from the same test should run tests.
+    //
+    maxInstances: 10,
```
Comment (Virtim) [inline wdio.conf.js]: change to 1 for now

File: wdio.conf.js [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    //
+    // If you have trouble getting all important capabilities together, check out the
+    // Sauce Labs platform configurator - a great tool to configure your capabilities:
+    // https://docs.saucelabs.com/reference/platforms-configurator
+    //
+    capabilities: [{
+        // maxInstances can get overwritten per capability. So if you have an in-house Selenium
+        // grid with only 5 firefox instances available you can make sure that not more than
+        // 5 instances get started at a time.
+        maxInstances: 5,
```
Comment (Virtim) [inline wdio.conf.js]: Again?

File: wdio.conf.js [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    //
+    // Warns when a deprecated command is used
+    deprecationWarnings: true,
+    //
+    // If you only want to run your tests until a specific amount of tests have failed use
+    // bail (default is 0 - don't bail, run all tests).
+    bail: 0,
+    //
+    // Saves a screenshot to a given path if a command fails.
+    screenshotPath: './errorShots/',
```
Comment (Virtim) [inline wdio.conf.js]: Comment it out for now

File: wdio.conf.js [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    // before running any tests.
+    framework: 'cucumber',
+    //
+    // Test reporter for stdout.
+    // The only one supported by default is 'dot'
+    // see also: http://webdriver.io/guide/reporters/dot.html
+    // reporters: ['dot'],//
+    // If you are using Cucumber you need to specify the location of your step definitions.
+    cucumberOpts: {
+        require: ['./support/steps'],        // <string[]> (file/dir) require files before executing features
```
Comment (Virtim) [inline wdio.conf.js]: require: ['./step-definitions/*.js', './step-definitions/when.js', './step-definitions/then.js'],

And Given :)

Comment (rosnovsky) [reply wdio.conf.js]: what's next?

**Pull request merged**